src/client/visualizations/line-chart/line-chart.tsx

```diff
@@ -37,7 +37,11 @@ export class LineChart extends BaseVisualization<BaseVisualizationState> {
   protected renderInternals(dataset: Dataset): JSX.Element {
     const { essence, timekeeper, stage } = this.props;
 
-    const scale = calculateXScale(essence, timekeeper, dataset, stage.width - Y_AXIS_WIDTH);
+    const range = calculateXRange(essence, timekeeper, dataset);
+    if (!range) {
+      return null;
```

Comment:
@piter75 @mkuthan What we really should do here? We can't create x-scale, so we can't show any line charts components. We also need to unmount `Interactions` component, because it has global event listeners on mouse move, which uses closed over values of x-scale. So we need to return something different? But returning null is weird, user would see blank screen. Maybe some warning? "No data for requested query" or something.

Reply:
I would display "No data" placeholder. If there are straightforward hints for the user what to do to get some data they could be shown as well.